lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -114,6 +114,9 @@ IPV_IDENTITY_PENDING_PAGE:
   response:
     type: page
     pageId: page-ipv-pending
+  events:
+    next:
+      targetState: PYI_F2F_DELETE_PAGE
```

Comment:
Is there a way we can put this behind the `deleteDetailsEnabled` feature flag so that we can test it (and write e2e tests) before we go live? I'm thinking maybe a copy of the `IPV_IDENTITY_PENDING_PAGE` that just has a continue button, that we route to when the flag is enabled. Could use a dynamic context for it